src/main/java/cis2910c/Main.java

```diff
@@ -0,0 +1,278 @@
+		
+		//loop to extract the indexes and combine the indexes with duplicate conditions
+		//to add to new list (i.e. -> {back[0], arm[1], leg[2,3]} )
+		for (int i = 0; i < disabilityCondition.size(); i++) {
+		    indexes.computeIfAbsent(disabilityCondition.get(i), c -> new ArrayList<>()).add(i);
+		}
+		
+		//add each bilateral to new list
+		indexes.forEach((k, v) -> {	
+			if(v.size() > 1) {
+				
+				for(int i = 0; i < v.size(); i++) {
+					//add to list to remove
+					listToRemove.add(disabilityList.get(v.get(i))); 
+					
+					/* 
+					 * create a new DisabilityRating object and add to tempList. This tempList will use the 
+					 * existing percent disability from the disabilityList indexed item, and then add the 
+					 * key "k" as the new "condition" that is disabled.
+					 * 
+					 * The new modified object is to combine the bilateral conditions into one new combined
+					 * disability where the percentages have been combined with the bilateral calculations. 
+					 * The tempList will be added to the "bilateral" list to calculate the bilateral rating. 
+					 *  
+					 * Example:------------
+					 * 
+					 * Original: 
+					 * 20% - left leg
+					 * 20% - right leg
+					 * 
+					 * New modified object:
+					 * 20% - leg
+					 * 20% - leg
+					 * 
+					 */
+					tempList.add(new DisabilityRating(disabilityList.get(v.get(i)).getPercent(), k));
+				}
+				//add tempList to bilateral list
+				bilateral.add(new ArrayList<>(tempList));
+				
+				//clear the tempList list so it does not add previous added disability lists
+				tempList.clear();
+			}
+		});
+		
+		//remove all bilateral items from disabilityList
+		disabilityList.removeAll(listToRemove);
+		
+		//do bilateral calculation if duplicate disabilities
+		calculateBilateral(bilateral, disabilityList);	
+		
+	}
+	
+	private static void calculateBilateral(List<ArrayList<DisabilityRating>> bilateral, ArrayList<DisabilityRating> disabilityList) {
+		
+		double efficiencyRating = 100.00;
+		double newDisabilityRating = 0;
+		String condition = null;
+		double newEfficiencyRating;
+		final double bilateralPercentAmount = .10;
+		
+		for(int i = 0; i < bilateral.size(); i++) {
+			
+			//method to sort from highest percentage to lowest
+			sortDisabilityPercentage(bilateral.get(i));
+			
+			newEfficiencyRating = efficiencyRating;
+			
+			for(int j = 0; j < bilateral.get(i).size(); j++) {
+				
+				condition = bilateral.get(i).get(j).getCondition();
+				newDisabilityRating = Math.round(newEfficiencyRating * ((double) bilateral.get(i).get(j).getPercent() / 100));
+				newEfficiencyRating = newEfficiencyRating - newDisabilityRating;
+			}
+			
+			newEfficiencyRating = efficiencyRating - newEfficiencyRating;
+			newEfficiencyRating = newEfficiencyRating + (newEfficiencyRating * bilateralPercentAmount);
+			newEfficiencyRating = Math.round(newEfficiencyRating);
+			
+			disabilityList.add(new DisabilityRating((int)newEfficiencyRating, condition));
```

Comment:
It looks like the bilateral rating of each unique set of bilateral conditions are being added to the final list.

According to the link we posted in chat: [va math]( https://militarydisabilitymadeeasy.com/va-math.html#bilateral)

All of the bilateral conditions are considered as one condition in the final list that includes non-bilateral conditions. 

It appears that multiple combined bilateral values will be added to the final list. Is my interpretation correct?

Reply:
Yes this is correct. If there is a bilateral condition, the disability percentage combines into a new higher percent by adding 10% to the total efficiency rating. For each bilateral condition, the math has to go through VA math cycle separately, then comes all together at the end as the final cycle. And also, the percentage needs to be in order from highest percent, to lowest percent, even after the combined new percent of the bilateral disability.

Please see screenshot in link to see the cycle https://pasteboard.co/ZSH1W0C1xdhM.png 
(this is coming from the PDF I uploaded. Sorry it this image turns out confusing).